src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -0,0 +1,160 @@
+
+  static get styles() {
+    const styles = styleElement.querySelector('template').content.textContent;
+    return css`
+      ${unsafeCSS(styles)}
+      input[type="radio"] {
+        display: none;
+      }
+      input[type="radio"]:checked + label.city-button {
+        background-color: rgba(255, 255, 255, 0.08);
+        box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.14), 0 2px 2px 0 rgba(0, 0, 0, 0.12), 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+        border-radius: 4px;
+        border: 1px solid var(--primary-green);
+      }
+      input[type="radio"] + label.city-button:hover {
+        border: 1px solid var(--primary-green);
+      }
+      input[type="radio"] + label.city-button {
+        display: inline-block;
+        flex: 1;
+        /* Distribute space evenly, accounting for margins, so there are always 4 cards per row. */
+        min-width: calc(25% - 24px);
+        position: relative;
+        margin: 4px;
+        padding-top: 24px;
+        text-align: center;
+        border: 1px solid;
+        border-color: rgba(0, 0, 0, 0);
+        cursor: pointer;
+        transition: 0.5s;
+        background: var(--background-contrast-color);
+        box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.14), 0 2px 2px 0 rgba(0, 0, 0, 0.12), 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+        border-radius: 4px;
+      }
+      input[type="radio"]:disabled + label.city-button {
+        /* TODO(alalama): make it look good and indicate disabled */
+        filter: blur(2px);
+      }
+      .city-name {
+        color: var(--light-gray);
+        font-size: 16px;
+        line-height: 19px;
+        padding: 48px 0 24px 0;
+      }
+      paper-button {
+        background: var(--primary-green);
+        color: #fff;
+        text-align: center;
+        font-size: 14px;
+      }
+      .flag {
+        width: 86px;
+        height: 86px;
+      }
+      .card-content {
+        display: flex;
+        flex-flow: wrap;
+        padding-top: 24px;
+      }
+      iron-icon {
+        color: var(--primary-green);
+        position: absolute;
+        top: 0;
+        right: 0;
+        margin: 6px;
+      }
+    `;
+  }
+
+  render() {
+    return html`
+    <outline-step-view display-action="">
+      <span slot="step-title">${this.localize('region-title')}</span>
+      <span slot="step-description">${this.localize('region-description')}</span>
+      <span slot="step-action">
+        <paper-button id="createServerButton" @tap="${this._handleCreateServerTap}" ?disabled="${!this._isCreateButtonEnabled(this.isServerBeingCreated, this.selectedLocationId)}">
+          ${this.localize('region-setup')}
+        </paper-button>
+      </span>
+      <div class="card-content" id="cityContainer">
+        ${this.locations.map(item => {
+          return html`
+          <input type="radio" id="card-${item.id}" name="${item.id}" ?disabled="${!item.available}" .checked="${this.selectedLocationId === item.id}" @tap="${this._locationSelected}">
```

Comment:
The name attribute is being misused. They should all have the same name as before. It defines the radio buttons set.

Reply:
Thanks for pointing that out